src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -378,7 +377,7 @@ public void canCreateARequestEvenWithDifferentCaseCheckedOutStatus() {
     loansFixture.checkOutByBarcode(smallAngryPlanet, rebecca);
 
     JsonObject itemWithChangedStatus = smallAngryPlanet.copyJson()
-      .put("status", new JsonObject().put("name", "Checked Out"));
+      .put("status", new JsonObject().put("name", "Checked out"));
```

Comment:
Why has this been changed? Is this still possible to do?

Reply:
You're right, it is not possible to do anymore. Because `inventory` does not allow such status.